specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2022-03-01-preview/streamingjobs.json

```diff
@@ -0,0 +1,1126 @@
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/StreamingJobNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The streaming job was successfully stopped."
+          },
+          "202": {
+            "description": "The stop streaming job request was successfully initiated."
+          },
+          "default": {
+            "description": "Error.",
+            "schema": {
+              "$ref": "../../common/v1/definitions.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.StreamAnalytics/streamingjobs/{jobName}/scale": {
+      "post": {
+        "tags": [
+          "StreamingJobs"
+        ],
+        "operationId": "StreamingJobs_Scale",
+        "description": "Scales a streaming job when the job is running.",
+        "x-ms-examples": {
+          "Scale a streaming job": {
+            "$ref": "./examples/StreamingJob_Scale.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "scaleJobParameters",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/ScaleStreamingJobParameters"
+            },
+            "description": "Parameters applicable to a scale streaming job operation."
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/StreamingJobNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The scale streaming job request was successfully initiated."
+          },
+          "default": {
+            "description": "Error.",
+            "schema": {
+              "$ref": "../../common/v1/definitions.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.StreamAnalytics/streamingjobs/{jobName}/skus": {
+      "get": {
```

Comment:
>get [](http://example.com/codeflow?start=7&length=3)

Recommend adding x-ms-pageable. Even if your API doesn't support pagination, you can use null for nextLinkName. This allows you to add pagination later without a new API version.